.kokoro/lint/continuous_queue.cfg

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Tell the trampoline which build file to use.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: ".kokoro/tests/run_tests_diff_head.sh"
```

Comment:
Continuous tests use `".kokoro/tests/run_tests_diff_head.sh"`, but I believe this is for presubmits - should we mimic `presubmit.cfg` and use `".kokoro/tests/run_tests_diff_main.sh"` accordingly?

Reply:
I think diff_main is more suitable for merge queue tests